js/gloss-witness.js

```diff
@@ -1431,11 +1471,11 @@ async function getAllWitnessFragmentsOfSource(fragmentSelections=null){
             return false
           }
       }
-
+    const sourceValue = sourceURI ? sourceURI : sourceHash ? sourceHash : null
     // Each source annotation targets a Witness.
     // Get all the source annotations whose value is this source string (URI or text string)
     const sourceAnnosQuery = {
-        "body.source.value": httpsIdArray(sourceURI),
+        "body.source.value": sourceValue.startsWith("http") ? httpsIdArray(sourceValue) : sourceValue,
```

Comment:
For what it's worth, the `httpsIdArray(val)` will just give back `[val]` if it isn't "http" anyway